lib/please_cli/please_cli/decision_task.py

```diff
@@ -176,21 +176,18 @@ def get_deploy_task(index,
         ]
 
     elif deploy_target == 'DOCKERHUB':
-        docker_repo = please_cli.config.DOCKER_REPO
         project_name = (
             '{project} ({nix_path_attribute}) to DOCKERHUB '
-            '({docker_repo}:{project}-{nix_path_attribute}-{channel})'.format(
+            '(tag {project}-{nix_path_attribute}-{channel})'.format(
                 project=project,
                 nix_path_attribute=nix_path_attribute,
-                docker_repo=docker_repo,
                 channel=channel,
             )
         )
         command = [
             './please', '-vv', 'tools', 'deploy:DOCKERHUB', project,
             '--taskcluster-secret=' + taskcluster_secret,
             '--nix-path-attribute={}'.format(nix_path_attribute),
-            '--docker-repo={}'.format(docker_repo),
```

Comment:
Actually passing arguments to `deploy:X` command is done intentionally and helped me (and others) in the past to debug, understand and remember what is happening when deploying.

I would rather explicitly state where do we deploy via command line arguments then to "hide" this configuration.

Lets rather add `docker_repo` and `docker_registry` to `lib/please_cli/please_cli/config.py` and pass the here as arguments.